packages/cascader/src/index.vue

```diff
@@ -504,6 +504,7 @@ export default defineComponent({
 
       if (isPromise(passed)) {
         passed.then(calculateSuggestions)
+          .catch(() => { /* prevent log error */ })
```

Comment:
Why should we hide error from bubbling up to the users?

Reply:
I personally think that the error should always be handled from the upper layer instead of us.

Reply:
@JeremyWuuuuu 

Users can catch that error in the corresponding callback too:

https://codesandbox.io/s/element-plus-spa-forked-jfhlz?file=/src/App.vue:290-321

In addition, IMHO, user have to write more code (in `onErrorCaptured` hook) to catch that the error bubbled from component, and that would be annoying.

Reply:
Well, point made. If we bubbles the error to the user layer the whole application crashes, which is not something we would like to see.